validphys2/src/validphys/kinematics.py

```diff
@@ -12,9 +12,16 @@
 from reportengine import collect
 from reportengine.checks import check_positive
 from reportengine.table import table
+
+
 from validphys.core import CutsPolicy
 from validphys.plotoptions import core as plotoptions_core
 
+
+
+
+
+
```

Comment:
this module should probably not be touched at all

Reply:
I agree, could you remove the new lines?